aptos-move/block-executor/src/view.rs

```diff
@@ -621,8 +648,13 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> ValueToIden
         let id = T::Identifier::try_from_move_value(layout, identifier_value, &())
             .map_err(|e| TransformationError(e.to_string()))?;
         match &self.latest_view {
+            // TODO are we at this point always supposed to read a value this transaction committed (i.e. AfterCurrentTxn)?
```

Comment:
makes sense, why TODO?

Reply:
to recheck with you. now I can remove :)